manifests/repositories.pp

```diff
@@ -10,9 +10,9 @@
   $apt_key_id      = $htcondor::apt_key_id
   $apt_key_source  = $htcondor::apt_key_source
   $condor_priority = $htcondor::condor_priority
-  $major_release   = regsubst($::operatingsystemrelease, '^(\d+)\.\d+$', '\1')
+  $major_release   = regsubst($facts['os']['name'], '^(\d+)\.\d+$', '\1')
```

Comment:
I think that should be `$facts['os']['release']['full']` or maybe drop the `regsubst` and use `$facts['os']['release']['major']`?